### ISSUE TITLE
Allow targetting other triples

### ISSUE THREAD

**Issue by Aareon** [syegulalp/Akilang#4]
https://github.com/syegulalp/Akilang/blob/270037f7c94873b4451f5b0af46f9d0b1764bbf2/aki/core/compiler.py#L17

**Comment by syegulalp** [comment]
Yes, that's a long-term goal. Eventually there will be a way to specify the target in a program's metadata (there's going to be a `meta` keyword reserved for that), and once that's in it shouldn't be hard to provide retargeting the output for another triple.
